ddtrace/tracer.py

```diff
@@ -312,6 +313,12 @@ def deregister_on_start_span(self, func):
     def debug_logging(self):
         return log.isEnabledFor(logging.DEBUG)
 
+    def global_excepthook(self, tp, value, traceback):
+        """The global tracer except hook."""
+        if asbool(os.getenv("DD_INSTRUMENTATION_TELEMETRY_ENABLED", True)):
```

Comment:
Can we use the config object to store a reference to this environment variable. Something like this: https://github.com/DataDog/dd-trace-py/blob/v1.12.1/ddtrace/settings/config.py#L228